src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -101,8 +101,31 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->registerSecurityCsrfConfiguration($config['csrf_protection'], $container, $loader);
 
+        $oldAssetsConfigured = false;
+        if (
+            isset($config['templating'])
+                &&
+            (
+                count($config['templating']['packages'])
+                    ||
+                count($config['templating']['assets_base_urls']['http'])
```

Comment:
I would use ``!empty(...)``, which would also remove the need for the initial ``isset($config['templating'])``